receiver/otlpreceiver/internal/arrow/arrow_test.go

```diff
@@ -719,6 +720,38 @@ func TestHeaderReceiverNoIncludeMetadata(t *testing.T) {
 	}
 }
 
+func TestHeaderReceiverAuthServerNoIncludeMetadata(t *testing.T) {
```

Comment:
quibble: could we rename this test to be `TestHeaderReceiverAuthServerIsSetNoIncludeMetadata` or something similar? initially my brain read this as testing the header receiver's auth server when include metadata is false and not testing that when the auth server isn't null and include is false that we indeed receive the headers expected.